setup.cfg

```diff
@@ -117,10 +117,12 @@ labels = lint
 skip_install = true
 allowlist_externals = bash
 commands =
-    bash -c "find . \( -name .git -o -name gradlew -o -name .tox \) -prune -o -print0 | xargs -0 file -N | grep shell.script | cut -f1 -d: | xargs shellcheck"
+    bash -c "find . \( -name .git -o -name gradlew -o -name .tox -o -name .ruff_cache -o \
+    -path ./tests/spread/tools/snapd-testing-tools \) -prune -o -print0 | xargs -0 file -N \
+    | grep shell.script | cut -f1 -d: | xargs shellcheck"
 
-[testenv:spellcheck]
-description = Check spelling using spread-shellcheck.py
+[testenv:spread-shellcheck]
```

Comment:
Note, the script `tools/spread-shellcheck.py` still checks and fails on the `./tests/spread/tools/snapd-testing-tools/` directory.

The script will need a bit of modification to ignore entire directories, but that can be a separate PR.